src/get-started/new-typography-scale/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: layout-pane.njk
+title: Updating your service to use the new typography scale
+section: Get started
+theme: How to guides
+order: 6
+description: The GOV.UK Design System team have been working on updating our typography scale provided with GOV.UK Frontend. This new scale is now available to use and will be enforced by default in GOV.UK Frontend version 6.0.0. Here's how to use it, what's changed and what to expect when implementing it into your service.
+---
+
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+The GOV.UK Design System team have been working on updating our typography scale provided with GOV.UK Frontend. This new scale is now available to use and will be enforced by default in GOV.UK Frontend version 6.0.0. Here's how to use it, what's changed and what to expect when implementing it into your service.
+
+## Changes we've made
+The new typography scale includes the following changes from the previous scale:
+
+{{ govukTable({
+    caption: "Changes to the typography scale",
+    captionClasses: "govuk-table__caption--m",
+    head: [
+        {
+            text: "Typography scale value"
```

Comment:
On the type scale page in the styles section, we call this "Point on type scale". I don't know which I prefer tbh! Any opinions?

Reply:
I don't have an opinion on which one tbh

Reply:
Decision: go with 'point on type scale'!

Reply:
```suggestion
            text: "Point on type scale"
```